src/vivarium_csu_hypertension_sdc/components/builder.py

```diff
@@ -131,16 +138,18 @@ def write_ckd_data(artifact, location):
 
     # Measures for Disease Model
     key = f'cause.chronic_kidney_disease.cause_specific_mortality_rate'
-    write(artifact, key, load(key))
+    csmr = load(key)
+    write(artifact, key, csmr)
 
     # Measures for Disease States
     key = 'cause.chronic_kidney_disease.prevalence'
-    write(artifact, key, load(key))
+    prevalence = load(key)
+    write(artifact, key, prevalence)
 
     # TODO: Find source for YLDs at the draw level to back calc disability weight.
 
-    key = 'cause.chronic_kidney_disease.excesss_mortality_rate'
-    write(artifact, key, load(key))
+    key = 'cause.chronic_kidney_disease.excess_mortality_rate'
+    write(artifact, key, csmr/prevalence)
```

Comment:
I think you want the post division clean up we have in vivarium_inputs (e.g., fill in na, inf) 
https://github.com/ihmeuw/vivarium_inputs/blob/release/3.1.0/src/vivarium_inputs/core.py#L171